### ISSUE TITLE
Accessibility module "Cannot read property 'series' of undefined" when tabbing chart when no data

### ISSUE THREAD

**Issue by venky1090** [highcharts/highcharts#10312]
#### Expected behaviour
Should be no error when tab through the chart with no data and accessibility module on.
There is should be no extra tab stop. Screen should just announce no data and go to next active element.

#### Actual behaviour
1. There is error in console.
2. There is tab stop, leading to WCAG issue

#### Live demo with steps to reproduce
http://jsfiddle.net/aq97025b/3/
Focus on 'link at top' and tab through
next tab -> overall chart highlights
next tab -> tab stop (internally, it's on ```<div tabindex="0" style="position: absolute; top: -999em; width: 1px; height: 1px; overflow: hidden;"></div>```)
next tab -> 'link at bottom'

Image
![image](https://user-images.githubusercontent.com/5780845/54347795-bc893900-466d-11e9-9cd4-d2bd6a2a7ba7.png)

#### Product version
Highcharts 6.1.3 and latest

#### Affected browser(s)
All

**Comment by sebastianbochan** [comment]
Hi @venky1090,
Thank you for reporting about the problem.

FYI @oysteinmoseng

**Comment by oysteinmoseng** [comment]
This will be fixed in Highcharts v7.1, to be released in a few weeks
